### PR TITLE
Adding 2 new conf to FastClick

### PR DESCRIPTION
**Pull request by lidall**
A pktgen script in order to generate traffic and receive them after passed through the DUT.
Besides, a single core NAT with DPDK enabled.

### CODE REVIEW

File: conf/nat/nat-DPDK.click [hunk 1]
```diff
@@ -0,0 +1,91 @@
+/*
+ * This is a script based on Thomar NAT and with DPDK enabled. Also you can  
+ * just replace the FromDPDKDevice and ToDPDKDevice with FromDevice and ToDevice
+ * if you don't want to implement DPDK.
```
Comment (tbarbette) [inline conf/nat/nat-DPDK.click]: to use DPDK

File: conf/nat/nat-DPDK.click [hunk 2]
```diff
@@ -0,0 +1,91 @@
+/*
+ * This is a script based on Thomar NAT and with DPDK enabled. Also you can  
+ * just replace the FromDPDKDevice and ToDPDKDevice with FromDevice and ToDevice
```
Comment (tbarbette) [inline conf/nat/nat-DPDK.click]: Queue -> ToDevice

File: conf/nat/nat-DPDK.click [hunk 3]
```diff
@@ -0,0 +1,91 @@
+/*
```
Comment (tbarbette) [inline conf/nat/nat-DPDK.click]: Please change filename to be lowercase

**Comment by Mr-Hongyi** [comment]
Actually, the pktgen-l3-latest.click is based on your script tester-l3.click. So you mean we just need to modify it and use the same filename to push?

**Comment by tbarbette** [comment]
Yes please

**Comment by Mr-Hongyi** [comment]
I think we are done and you can check it (:

**Comment by tbarbette** [comment]
Thanks for the PR. Please look at my first correction commit though ;)

Things like NDESC, you can change it locally by overwriting ring_size but it's best to keep the default for examples.

**Pull request merged**